### PR TITLE
https://github.com/coala/coala/issues/3212

### PR DESCRIPTION
**Pull request by sils**
https://github.com/coala/coala/issues/3212

### CODE REVIEW

**Comment by Mixih** [comment]
Not exactly sure how we're going to test this as all run_coala options are tests as parts of coala-modes. Do we need a new mode, or should we write a test for this file?

File: coalib/coala_main.py [hunk 1]
```diff
@@ -91,7 +94,9 @@ def run_coala(console_printer=None,
             if not section.is_enabled(targets):
                 continue
 
-            if force_show_patch:
+            if not autoapply:
+                section['default_actions'] = ''
+            elif force_show_patch:
                 section['default_actions'] = '*: ShowPatchAction'
```
Comment (Mixih) [inline coalib/coala_main.py]: I think you need to make it clear in the doc string that `autoapply` and `force_show_patch` cannot be used together

File: coalib/coala_main.py [hunk 2]
```diff
@@ -49,6 +50,9 @@ def run_coala(console_printer=None,
     :param nothing_done:            A callback that will be called with only a
                                     log printer that shall indicate that
                                     nothing was done.
+    :param autoapply:               Set this to false to not autoapply any
+                                    actions. If you set this to `True`,
```
Comment (Mixih) [inline coalib/coala_main.py]: I think you mean `False`

**Comment by Mixih** [comment]
ack 06a772f

**Comment by Mixih** [comment]
ack cb1c7d

**Comment by sils** [comment]
@rultor merge

**Pull request merged**